plan/expression_rewriter.go

```diff
@@ -274,7 +274,7 @@ func (er *expressionRewriter) Enter(inNode ast.Node) (ast.Node, bool) {
 
 func (er *expressionRewriter) handleWindowFunction(v *ast.WindowFuncExpr) (ast.Node, bool) {
 	er.p = er.b.buildWindowFunction(er.p, v)
```

Comment:
Let `buildWindowFunction` return `WindowFunction` plan, so we can avoid type assertion.